src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
   api:
     # on the verge of every deployment (fortnightly), update the "image" value from the "api" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json
-    image: daschswiss/knora-api:29.1.3
+    image: daschswiss/knora-api:v30.0.0
```

Comment:
`knora-sipi` (line 31) must always have the same value than `knora-api`. We need to bump them both to the same version number

Reply:
ok then i misunderstood that